src/rules.js

```diff
@@ -1841,6 +1841,39 @@ module.exports = [
     },
     /**
      ==================================================================
+     TEXT-INDENT
+     ==================================================================
+     */
+    {
+        type: 'pattern',
+        name: 'Text indent',
+        prefix: 'Ti-',
+        properties: ['text-indent'],
+        allowCustom: true,
+        allowSuffixToValue: true,
+        rules: [
+            {suffix: 'inh', values: ['inherit']}
+        ]
+    },
+    /**
+     ==================================================================
+     TEXT-OVERFLOW
+     ==================================================================
+     */
+    {
+        type: 'pattern',
+        name: 'Text overflow',
+        prefix: 'Tov-',
+        properties: ['text-overflow'],
+        allowCustom: true,
+        allowSuffixToValue: false,
+        rules: [
+            {suffix: 'c', values: ['clip']},
```

Comment:
Any thoughts on adding support for left overflow?  (eg, `text-overfow: clip ellipses`, or `text-overflow: clip clip`, etc)